PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3278,6 +3278,12 @@ void Notepad_plus::docOpenInNewInstance(FileTransferMode mode, int x, int y)
 	command += pX;
 	command += TEXT(" -y");
 	command += pY;
+	command += TEXT(" -l");
+	command += ScintillaEditView::langNames[buf->getLangType()].lexerName;
+	command += TEXT(" -n");
+	command += to_wstring(_pEditView->getCurrentLineNumber() + 1);
+	command += TEXT(" -c");
+	command += to_wstring(_pEditView->getCurrentColumnNumber() + 1);

```

Comment:
1. Do anyone know if this is the best conversion from Long type we can use here? (`to_wstring`)
2. Above, in lines 3275/3276 is it trying to get the current window coordinates (x/y)? Because I think it doesn't open new instance in the same coordinates of current window, look like new window is always opened at 0,0.